ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.15.yaml

```diff
@@ -241,6 +251,15 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-mag-ipi-fullyprivate
+- as: azure-upi-custom-nsg-f14
```

Comment:
How about `f28` based on we also have azure-ipi to cover it.

Reply:
sure, updated.